CanvasAPI/CanvasAPI.py

```diff
@@ -151,3 +152,97 @@ def getCoursesAssignmentsObjects(self, courseID):
                 .jsonObjects(object_hook=self.jsonObjectHook)
 
         return coursesAssignments
+
+    def getCoursesUsers(self, courseID, enrollmentType=None, **kwargs):
+        """
+        Get Canvas Users objects as requests Response object.  May be one of multiple pages.
+
+        :param courseID: ID number of the Canvas course object's User Objects to be retrieved
+        :type courseID: int
+        :param enrollmentType:
+        :type enrollmentType: str
+        :return:
+        :rtype: requests.models.Response
+        """
+        assert type(courseID) is int
+
+        if enrollmentType:
+            if not isinstance(enrollmentType, str):
+                raise TypeError('enrollmentType must be string')
+            if 'enrollment_type' not in kwargs:
+                kwargs['enrollment_type'] = str(enrollmentType)
+
+        queryURI = self._QueryURIs.COURSES_USERS.format(courseID=courseID)
+        response = self.get(queryURI, params=kwargs)
+
+        return response
+
+    def getCoursesUsersObjects(self, courseID, enrollmentType=None, **kwargs):
```

Comment:
getCourseUsers and getCourseUsersObjects seems a little bit ambiguous. I see that the getCourseUsers function returns a response. Would it be more clear to rename the function getCourseUsersResponse?